hexa/workspaces/schema/mutations.py

```diff
@@ -62,6 +62,8 @@ def resolve_update_workspace(_, info, **kwargs):
         if input.get("description", None):
             args["description"] = input["description"]
 
+        args["docker_image"] = input.get("dockerImage", "")
```

Comment:
This is different than the other cases around this one where we check if there is an input given in order to avoid to override the dockerImage when it's not given explicitly by the user